app/src/main/java/org/wikipedia/util/FeedbackUtil.java

```diff
@@ -173,6 +176,19 @@ public static Toast showToastOverView(View view, CharSequence text, int duration
         return toast;
     }
 
+    @SuppressWarnings("checkstyle:magicnumber")
+    public static Balloon showTooltipBubble(@NonNull Context context, @NonNull CharSequence text) {
```

Comment:
Why call it a "bubble" when the library calls it a "balloon"? Can we just say `showTooltip()`?